indra/llcommon/llapp.cpp

```diff
@@ -414,6 +414,22 @@ void LLApp::writeMiniDump()
 {
 }
 
+// static
+void LLApp::quit(bool force)
```

Comment:
That looks nice. But shouldn't any other `app->forceQuit()` and `app->requestQuit()` be replaced with the wrapper as well?

Reply:
They could, but not should :-)